dash_bio/AlignmentChart.py

```diff
@@ -92,13 +92,16 @@ class AlignmentChart(Component):
 if either of them is set.
 - height (number | string; optional): Width of the Viewer.
 Property takes precedence over tilesheight if both
-are set."""
+are set.
+
+Available events: """
```

Comment:
I think you might be using the wrong version of Dash to build these. Have you tried reinstalling the requirements from `requirements.txt`? Dash events were deprecated in version `0.36.0` (https://github.com/plotly/dash/blob/master/CHANGELOG.md#removed). This will get rid of these diffs.

Reply:
Oh, that's because I didn't commit / push after I rebuilt the package! Doing now.